entangler/Cargo.toml

```diff
@@ -22,13 +22,10 @@ serde_json = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
 
-[dependencies.storage]
-path = "../storage"
-features = []
+recall_entanglement_storage = { version = "0.1.0", path = "../storage", features = [] }
```

Comment:
adding version here should allow publishing to Crates and keeping the relative path for development.

Reply:
ah, nice... i always wondered why some cargo deps have version and path